client/payments/data/action-types.ts

```diff
@@ -0,0 +1,6 @@
+export enum ACTION_TYPES {
+	GET_PAYMENT_GATEWAYS_SUCCESS = 'GET_PAYMENT_GATEWAYS_SUCCESS',
+	UPDATE_PAYMENT_GATEWAY_SUCCESS = 'UPDATE_PAYMENT_GATEWAY_SUCCESS',
+	SET_ERROR = 'SET_ERROR',
```

Comment:
I think this can be removed in favor of `UPDATE_PAYMENT_GATEWAY_ERROR` and then handle adding the error in the reducer.

Reply:
I totally agree that `SET_ERROR` naming should be avoided, but it's being used in [updatePaymentGateway](https://github.com/woocommerce/woocommerce-admin/blob/add/6912-date-store-for-payment-gateways/client/payments/data/actions.ts#L86), [getPaymentGateway](https://github.com/woocommerce/woocommerce-admin/blob/add/6912-date-store-for-payment-gateways/client/payments/data/resolvers.ts#L29), and [getPaymentGateway](https://github.com/woocommerce/woocommerce-admin/blob/add/6912-date-store-for-payment-gateways/client/payments/data/resolvers.ts#L29).

If we create an action creator and type for each, we end up with a duplicate code. Do you have a preferred way to handle duplication in this case?

Reply:
Well, I think it should be okay since we only have three functions. I will make an update.

Reply:
updated in https://github.com/woocommerce/woocommerce-admin/pull/6918/commits/429dbb5635fd656295a399eba291896d77989e74